install/install.sh

```diff
@@ -41,13 +41,15 @@ usage() {
     * Specify a database management system. Valid options include: postgres, sqlite (if not specified, the default is to use sqlite).
       If the database management system is specified to use postgres, the following extra variables must also be set: QPC_DBMS_HOST, QPC_DBMS_PASSWORD
          -e QPC_DBMS=sqlite
+    * Specify if you would like to use external preconfigured PostgresSQL.
+         -e EXTERNAL_POSTGRES=true
     * If using postgres, specify the db name (if not specified the default value is postgres):
          -e QPC_DBMS_DATABASE=postgres
     * If using postgres, specify the db user (if not specified the default value is postgres):
          -e QPC_DBMS_USER=postgres
-    * If using postgres, you must specify the db password:
+    * If using postgres, you must specify the db password if EXTERNAL_POSTGRES = true (otherwise default value is password):
```

Comment:
If we have an external postgres set up, they must set the password unless it also happens to be "password". If they are using the internal postgres, do they still have the option to create their own password? If so, it needs to be clear